libcxx/test/libcxx/containers/sequences/forwardlist/bool-conversion.pass.cpp

```diff
@@ -0,0 +1,36 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Please add `// REQUIRES: std-at-least-c++20` to both new test files. Otherwise, these tests meaninglessly run and fail in C++03/11/14/17 modes.